packages/eslint-plugin-fiori-tools/eslintrc-typescript.js

```diff
@@ -5,7 +5,17 @@ const overrides = [
     {
         'plugins': ['@typescript-eslint'],
         'files': [`./${sourceCodePath}/*.ts`, `./${sourceCodePath}/**/*.ts`],
-        'excludedFiles': ['*.d.ts', '**/*.d.ts'],
+        'excludedFiles': [
+            'target/**',
+            `${sourceCodePath}/test/changes_loader.ts`,
+            `${sourceCodePath}/test/changes_preview.ts`,
```

Comment:
Why do we exclude these ones? Same for `${testCodePath}/**/Example.qunit.ts`

Reply:
These test files have eslint errors. But in some cases I think it we better clarity of example to keep these issues. `reject` for example. 

What do you think?

`changes_loader.ts`
```
open-ux-tools/packages/fiori-elements-writer/test/test-output/lropTemplates/lropV2_ts_ui5_1_108/webapp/test/changes_loader.ts
      16:50  error  'reject' is defined but never used  @typescript-eslint/no-unused-vars
      34:36  error  'obj' is defined but never used     @typescript-eslint/no-unused-vars
```

`changes_preview.ts`    
```
open-ux-tools/packages/fiori-elements-writer/test/test-output/lropTemplates/lropV2_ts_ui5_1_108/webapp/test/changes_preview.ts
        9:31  error  'oChange' is defined but never used      @typescript-eslint/no-unused-vars
       48:55  error  'reject' is defined but never used       @typescript-eslint/no-unused-vars
       82:76  error  'rejectInner' is defined but never used  @typescript-eslint/no-unused-vars
      109:61  error  'obj' is defined but never used          @typescript-eslint/no-unused-vars
```



`Example.qunit.ts` 
```   
open-ux-tools/packages/ui5-library-writer/test/test-output/com.sap.myui5tslib113/test/com/sap/myui5tslib113/qunit/Example.qunit.ts
      32:40  error  'reject' is defined but never used             @typescript-eslint/no-unused-vars
      54:9   error  'oExample' is assigned a value but never used  @typescript-eslint/no-unused-vars
      60:40  error  'reject' is defined but never used             @typescript-eslint/no-unused-vars
```

Reply:
I would have recommended to rename them to `_reject` etc. however, since we are going to get rid of these files anyhow (hopefully sooner than later) but still some projects have them, I understand why it is more convenient for consumers if we simply ignore them.